account/identity.go

```diff
@@ -125,3 +126,15 @@ func (m *GormIdentityRepository) Query(funcs ...func(*gorm.DB) *gorm.DB) ([]*Ide
 	}
 	return objs, nil
 }
+
+// List return all user identities
+func (m *GormIdentityRepository) List(ctx context.Context) ([]*app.Identity, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "list"}, time.Now())
+	var objs []*app.Identity
+
+	err := m.db.Table(m.TableName()).Find(&objs).Error
+	if err != nil && err != gorm.ErrRecordNotFound {
+		return nil, err
```

Comment:
Does this mean there are zero records ? If yes, it should return an empty list instead of an `err` .